airflow/operators/dagrun_operator.py

```diff
@@ -51,15 +51,16 @@ class TriggerDagRunOperator(BaseOperator):
     def __init__(
             self,
             trigger_dag_id,
-            python_callable,
+            python_callable=None,
             *args, **kwargs):
         super(TriggerDagRunOperator, self).__init__(*args, **kwargs)
         self.python_callable = python_callable
         self.trigger_dag_id = trigger_dag_id
 
     def execute(self, context):
         dro = DagRunOrder(run_id='trig__' + timezone.utcnow().isoformat())
-        dro = self.python_callable(context, dro)
+        if self.python_callable:
```

Comment:
Can we compare explicitly against `None`?

i.e.: `if self.python_callable is not None:`

It's not hugely common but it is entirely possible to have a Python object that is both callable and falsey.

Reply:
Ok, updated both of these.